client/app/bundles/shared/components/Tag.scss

```diff
@@ -4,14 +4,16 @@
   padding: 5px 12px;
   text-indent: 4px;
   display: inline-block;
+  margin-right: 5px;
```

Comment:
@Seunadex Sure, I'm just using the word "sibling" to describe tags that are next to other tags! You'll have to change the CSS selector slightly to get it to work since tags are wrapped in spans, though.